docs/proposals/permissive-mtl.md

```diff
@@ -0,0 +1,42 @@
+# Permissive mTLS mode
+
+## Context 
+
+The goal for this proposal is to provide a more pragmatic way to enable mTLS in a service mesh. When working with existing applications, 
+it is not practical to have mTLS enabled for everything or nothing, but we need to provide a more gradual way to do it.
+Therefore, a "permissive" mTLS mode (as opposed to the "strict" one we use today) would allow a team to enable mTLS yet allow 
+traffic that is not part of the zero-trust infrastructure to still be able to make requests without a strict validation of the 
+certificates on incoming requests.
```

Comment:
Can we get a more concrete definition of what "permissive" means? Normally in TLS conversations, it means you have a TLS session that opts out of certain certificate verifications. In a mTLS context, I am expecting it to mean that TLS is being used but we either the server of the client is not being verified.

Reply:
yeh that's great.